app/mailers/subscription_mailer.rb

```diff
@@ -6,7 +6,18 @@ def confirmation(subscription_id)
     view_mail(
       NOTIFY_SUBSCRIPTION_CONFIRMATION_TEMPLATE,
       to: @subscription.email,
-      subject: I18n.t('job_alerts.confirmation.email.subject', reference: @subscription.reference),
+      subject: I18n.t('subscription_mailer.confirmation.subject'),
+    )
+  end
+
+  def update(subscription_id)
+    subscription = Subscription.find(subscription_id)
+    @subscription = SubscriptionPresenter.new(subscription)
+
+    view_mail(
+      NOTIFY_SUBSCRIPTION_CONFIRMATION_TEMPLATE,
```

Comment:
Should this be UPDATE_TEMPLATE as it is a different message and we distinguish NOTIFY_SUBSCRIPTION_DAILY_TEMPLATE / NOTIFY_SUBSCRIPTION_WEEKLY_TEMPLATE?

Reply:
Technically it should, but it is exactly the same and it is just a confirmation of create/update subscription

Reply:
The ticket for putting email content onto Notify is in this Sprint, so maybe it would help to be organised in this way. https://dfedigital.atlassian.net/browse/TEVA-1348

Reply:
Added the extra template